### PR TITLE
Clarify confidential client registration

### PR DESCRIPTION
**Pull request by daliboz**
Updating registration process for confidential clients.

### CODE REVIEW

File: content/dstu2/authorization.md [hunk 1]
```diff
@@ -68,11 +68,21 @@ resources and exchange context information with client applications.
...
 register applications. Obtaining a Cerner Care account is free and 
 requires nothing but an email address.
 
+If registering an application that will not be using offline_access, and that is not 
+going to be accessing services on behalf of a system (is not a confidential client), 
+your client application must first register using our 
+[code console][CERNER-CODE-CONSOLE]. 
+
+If registering a confidential client application (required for offline_access and
+access on behalf of a system), please first request an account via the **sandbox** instance of
```
Comment (daliboz) [inline content/dstu2/authorization.md]: I should switch this to non-production to match with the documentation on the wiki.

File: content/dstu2/authorization.md [hunk 2]
```diff
@@ -68,11 +68,21 @@ resources and exchange context information with client applications.
...
 requires nothing but an email address.
 
+If registering an application that will not be using offline_access, and that is not 
+going to be accessing services on behalf of a system (is not a confidential client), 
+your client application must first register using our 
+[code console][CERNER-CODE-CONSOLE]. 
+
+If registering a confidential client application (required for offline_access and
+access on behalf of a system), please first request an account via the **sandbox** instance of 
+[Cerner Central System Account Management][SYSTEM-ACCOUNTS]. Once the request is completed,
```
Comment (daliboz) [inline content/dstu2/authorization.md]: Should switch this over to the specific registration page: https://wiki.ucern.com/display/CernerCentral/Requesting+A+System+Account

File: content/dstu2/authorization.md [hunk 3]
```diff
@@ -68,11 +68,21 @@ resources and exchange context information with client applications.
 
 ## Registration ##
 In order for your client application to utilize any protected resources, 
-your client application must first register using our 
-[code console][CERNER-CODE-CONSOLE]. A Cerner Care account is required to 
+your client application must first register. A Cerner Care account is required to 
 register applications. Obtaining a Cerner Care account is free and 
 requires nothing but an email address.
 
+If registering an application that will not be using offline_access, and that is not
```
Comment (dennispatterson) [inline content/dstu2/authorization.md]: Can this be stated more positively, mentioning a provider or patient access workflow with a logged-in user?

Comment (daliboz) [inline content/dstu2/authorization.md]: offline_access is with a logged on user. Suggestions on wording to make it more clear? I was struggling with keeping it simple but easy to understand

Comment (dennispatterson) [inline content/dstu2/authorization.md]: "If registering an application that is using the public app profile and requires access only when the user is online,..."?

**Comment by daliboz** [comment]
addressed comments so far and squashed

**Comment by dennispatterson** [comment]
+1

**Comment by whitehatguy** [comment]
I made some [counter-proposals][1] in a pull request to clean up the grammar.

[1]: https://github.com/cerner/fhir.cerner.com/pull/111

**Comment by daliboz** [comment]
@whitehatguy They can't use it during code registration yet - it has to be done manually by us until that enhancement has been completed. We will update this document once that has been deployed.

Also, I would like to keep them close together, and ensure they don't go to the code console first, since they cannot start there for confidential apps. There is no way to modify apps right now, so if they started there for a confidential client, it would break/have to be deleted.

**Comment by whitehatguy** [comment]
So, I completely misunderstood.  How does someone get a Cerner contact if they are reading this guide for the first time?  Should we just guide individuals whom need confidential client access to reach out to a particular individual?

**Comment by daliboz** [comment]
As of right now, we're only piloting this with apps that already have a relationship with us (like the precision medicine app). It's too much manual work right now to make it available more broadly yet. We point them to this doc when they ask.

**Comment by whitehatguy** [comment]
That's fine - how do we indicate in the documentation that this is for "apps that already have a relationship with us" as to not confuse other developers?

**Comment by daliboz** [comment]
@whitehatguy look at latest and let me know what you think

**Comment by whitehatguy** [comment]
+1

**Pull request merged**